backend/config/application.rb

```diff
@@ -38,5 +38,8 @@ class Application < Rails::Application
         config.contract_template_dir =
             ENV.fetch('CONTRACT_TEMPLATE_DIR', '/storage/contract_templates')
                 .presence || '/storage/contract_templates'
+        
+        # This was added to use in the emails that send contract links.
+        config.base_url = ENV.fetch('BASE_URL', 'localhost:3000')
```

Comment:
`localhost:3000` is only the route during debug...it's a little worrysome seeing it show up here, but then again, normally it should be using `BASE_URL`